projects/packages/my-jetpack/_inc/hooks/use-notification-watcher/use-backup-needs-attention-notice.tsx

```diff
@@ -58,7 +58,11 @@ const useBackupNeedsAttentionNotice: NoticeHookType = ( redBubbleAlerts, isLoadi
 	}, [ recordEvent, status, contactSupportUrl ] );
 
 	useEffect( () => {
-		if ( ! redBubbleAlerts?.backup_failure || status === 'backups-deactivated' ) {
+		if (
+			! redBubbleAlerts?.backup_failure ||
+			status === 'backups-deactivated' ||
+			status === 'multisite_not_supported'
```

Comment:
This will prevent this notice to shows up:

<img width="1674" height="590" alt="image" src="https://github.com/user-attachments/assets/ee6f14e2-638b-4f73-9422-e1b43f17ffe4" />